README.md

```diff
@@ -3,7 +3,6 @@
 [![Build](https://github.com/ballerina-platform/module-ballerinax-twilio/actions/workflows/ci.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-twilio/actions/workflows/ci.yml)
 [![Trivy](https://github.com/ballerina-platform/module-ballerinax-twilio/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-twilio/actions/workflows/trivy-scan.yml)
 [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-twilio/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-twilio)
-[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-twilio/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-twilio/actions/workflows/build-with-bal-test-native.yml)
```

Comment:
I dont think we need to remove the compatibility badge, as we still have the on-demand workflow

Reply:
Yes. changed with [5110ab](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/213/commits/5110ab9890962ebb3f4e447b02615d7c00e6f841).